test/github_app_tool_db_dump.sql

```diff
@@ -170,8 +170,8 @@ workflows:
      primaryDescriptorPath: /workflows/HelloWorld.cwl
 ', 'DOCKSTORE_YML', '/.dockstore.yml', '2022-01-14 10:04:44.458000', '2022-01-14 10:04:44.458000', '/.dockstore.yml', false);
 
-INSERT INTO public.apptool (id, author, conceptdoi, dbcreatedate, dbupdatedate, description, email, giturl, ispublished, lastmodified, lastupdated, licensename, topicid, checkerid, descriptortype, descriptortypesubclass, forumurl, mode, organization, repository, sourcecontrol, workflowname, actualdefaultversion, topicautomatic, topicmanual, topicselection, waseverpublic) VALUES (50, null, null, '2022-01-12 14:51:23.656000', '2022-01-12 14:51:23.689000', null, null, 'git@github.com:C/github-app-tools.git', true, '2022-01-12 14:48:20.000000', '2022-01-12 14:51:22.707000', null, null, null, 'cwl', 'n/a', null, 'DOCKSTORE_YML', 'C', 'test-github-app-tools', 'github.com', 'testing', null, null, null, 'AUTOMATIC', true);
-INSERT INTO public.apptool (id, author, conceptdoi, dbcreatedate, dbupdatedate, description, email, giturl, ispublished, lastmodified, lastupdated, licensename, topicid, checkerid, descriptortype, descriptortypesubclass, forumurl, mode, organization, repository, sourcecontrol, workflowname, actualdefaultversion, topicautomatic, topicmanual, topicselection, waseverpublic) VALUES (51, 'testuser', null, '2022-01-12 14:45:55.370000', '2022-01-14 10:05:15.491000', 'test description...', 'testemail@gmail.com', 'git@github.com:C/github-app-tools.git', false, '2022-01-14 09:13:50.000000', '2022-01-12 14:45:54.113000', null, null, null, 'cwl', 'n/a', null, 'DOCKSTORE_YML', 'C', 'test-github-app-tools', 'github.com', 'md5sum', null, null, null, 'AUTOMATIC', true);
+INSERT INTO public.apptool (id, conceptdoi, dbcreatedate, dbupdatedate, description, giturl, ispublished, lastmodified, lastupdated, licensename, topicid, checkerid, descriptortype, descriptortypesubclass, forumurl, mode, organization, repository, sourcecontrol, workflowname, actualdefaultversion, topicautomatic, topicmanual, topicselection, waseverpublic) VALUES (50, null, '2022-01-12 14:51:23.656000', '2022-01-12 14:51:23.689000', null, 'git@github.com:C/github-app-tools.git', true, '2022-01-12 14:48:20.000000', '2022-01-12 14:51:22.707000', null, null, null, 'cwl', 'n/a', null, 'DOCKSTORE_YML', 'C', 'test-github-app-tools', 'github.com', 'testing', null, null, null, 'AUTOMATIC', true);
```

Comment:
Follow up to https://github.com/dockstore/dockstore/pull/5706, needed to drop the author and email column for tests to pass